test/tracker/ClickTracker.test.ts

```diff
@@ -201,6 +201,8 @@ describe('ClickTracker test', () => {
 
 	test('test add A tag and trigger MutationObserver', async () => {
 		clickTracker.setUp();
+		window.document.dispatchEvent(new window.Event('load'));
+		await sleep(100);
```

Comment:
Why sleep?

Reply:
because `window.onload = function () {}` is called asynchronously after the 'load' event dispatched, we should wait for it to be called.